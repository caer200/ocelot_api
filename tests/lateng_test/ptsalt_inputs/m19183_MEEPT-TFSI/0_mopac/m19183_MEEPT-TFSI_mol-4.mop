CHARGES

 
C	3.898841 +1 14.239799 +1 9.418596 +1
S	3.512200 +1 12.493442 +1 9.783603 +1
F	4.272334 +1 14.885266 +1 10.522023 +1
F	4.870535 +1 14.339232 +1 8.529815 +1
F	2.822405 +1 14.852348 +1 8.934233 +1
N	4.905299 +1 11.935022 +1 10.261173 +1
O	3.214000 +1 11.931289 +1 8.505139 +1
O	2.477822 +1 12.544686 +1 10.765358 +1
S	5.414607 +1 11.901765 +1 11.749453 +1
C	5.186747 +1 10.140306 +1 12.192958 +1
O	4.657355 +1 12.633938 +1 12.708466 +1
O	6.834726 +1 12.050575 +1 11.739108 +1
F	5.642197 +1 9.930750 +1 13.423303 +1
F	5.861682 +1 9.353326 +1 11.369559 +1
F	3.917124 +1 9.794666 +1 12.145662 +1
