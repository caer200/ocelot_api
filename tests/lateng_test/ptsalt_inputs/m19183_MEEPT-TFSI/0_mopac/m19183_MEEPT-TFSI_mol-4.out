 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 174**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Tue Oct  8 20:08:13 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          3.89884100  *  14.23979900  *   9.41859600  *
     2       S          3.51220000  *  12.49344200  *   9.78360300  *
     3       F          4.27233400  *  14.88526600  *  10.52202300  *
     4       F          4.87053500  *  14.33923200  *   8.52981500  *
     5       F          2.82240500  *  14.85234800  *   8.93423300  *
     6       N          4.90529900  *  11.93502200  *  10.26117300  *
     7       O          3.21400000  *  11.93128900  *   8.50513900  *
     8       O          2.47782200  *  12.54468600  *  10.76535800  *
     9       S          5.41460700  *  11.90176500  *  11.74945300  *
    10       C          5.18674700  *  10.14030600  *  12.19295800  *
    11       O          4.65735500  *  12.63393800  *  12.70846600  *
    12       O          6.83472600  *  12.05057500  *  11.73910800  *
    13       F          5.64219700  *   9.93075000  *  13.42330300  *
    14       F          5.86168200  *   9.35332600  *  11.36955900  *
    15       F          3.91712400  *   9.79466600  *  12.14566200  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         C          3.8988     14.2398      9.4186
     2         S          3.5122     12.4934      9.7836
     3         F          4.2723     14.8853     10.5220
     4         F          4.8705     14.3392      8.5298
     5         F          2.8224     14.8523      8.9342
     6         N          4.9053     11.9350     10.2612
     7         O          3.2140     11.9313      8.5051
     8         O          2.4778     12.5447     10.7654
     9         S          5.4146     11.9018     11.7495
    10         C          5.1867     10.1403     12.1930
    11         O          4.6574     12.6339     12.7085
    12         O          6.8347     12.0506     11.7391
    13         F          5.6422      9.9307     13.4233
    14         F          5.8617      9.3533     11.3696
    15         F          3.9171      9.7947     12.1457

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C2 N O4 S2 F6  =    15 atoms



      MOLECULAR POINT GROUP   :   C2  

   TOPOGRAPHY OF SYSTEM

   ATOM No.    LABEL   Atoms connected to this atom
      1          C      2      3      4      5
      2          S      7      8      6      1
      3          F      1
      4          F      1
      5          F      1
      6          N      2      9
      7          O      2
      8          O      2
      9          S     11     12      6     10
     10          C      9     13     14     15
     11          O      9
     12          O      9
     13          F     10
     14          F     10
     15          F     10

     Ion Atom No.  Type    Charge
 
    1       6      N       +1

          COMPUTED CHARGE ON SYSTEM: +1

          COMPUTED CHARGE ON SYSTEM: +1

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
